projects/angular/src/layout/tabs/tabs-actions.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, ContentChildren, ElementRef, HostBinding, Input, QueryList } from '@angular/core';
+
+import { ClrTabAction } from './tab-action.directive';
+
+export type ClrTabsActionsPosition = 'left' | 'right';
+
+@Component({
+  selector: 'clr-tabs-actions',
+  template: `
+    <div class="tabs-actions-wrapper">
+      <ng-content></ng-content>
+    </div>
+  `,
+  host: {
+    '[class.tabs-actions]': 'true',
+  },
+})
+export class ClrTabsActions {
+  @Input()
+  @HostBinding('attr.position')
+  position: ClrTabsActionsPosition = 'right';
+
+  @ContentChildren(ClrTabAction, { read: ElementRef }) actions: QueryList<ElementRef>;
```

Comment:
Is this used any where?

```suggestion
```

Reply:
Resolved in [2d0a306](https://github.com/vmware-clarity/ng-clarity/pull/1620/commits/2d0a306495298892d43ee3dd569062800ee44180)